translations/en.json

```diff
@@ -1482,6 +1482,7 @@
     "add-nickname": "Add a nickname (optional)",
     "nickname-description": "Nicknames help you identify others in Status.\nOnly you can see the nicknames you’ve added",
     "accept": "Accept",
+    "reply": "Reply",
```

Comment:
There is already a "Reply" translation line 869 "message-reply". Maybe delete one of them.

Reply:
Done :)